config/src/config/state_sync_config.rs

```diff
@@ -66,17 +66,17 @@ pub enum BootstrappingMode {
 /// bootstrapped and the blockchain continues to grow, e.g., continuously
 /// executing all transactions.
 #[derive(Copy, Clone, Debug, Deserialize, PartialEq, Serialize)]
-pub enum SyncingMode {
+pub enum ContinuousSyncingMode {
     ExecuteTransactions,     // Executes transactions to stay up-to-date
     ApplyTransactionOutputs, // Applies transaction outputs to stay up-to-date
 }
 
 #[derive(Copy, Clone, Debug, Deserialize, PartialEq, Serialize)]
 #[serde(default, deny_unknown_fields)]
 pub struct StateSyncDriverConfig {
+    pub activate_state_sync_v2: bool,          // Enables state sync v2
```

Comment:
These should be the final names (they match those of the modules used by the driver), but I've removed the commit anyway because it causes issues with backward compatibility tests.. lol 😆